README.md

```diff
@@ -69,6 +69,10 @@ Commands:
 | validate \| vv `<bundlePath>` | Validates a code bundle version without uploading the code bundle. |
 | create-auth-token \| auth `[options] <secretKey>` | Generates an authentication token that can be used to get detailed EdgeWorker debug response headers. |
 | generate-secret \| secret `[options]` | Generates a secret key that can be used to generate auth token or in property variable. |
+| clone \| clone `<edgeworker-identifier> <resourceTierId> [options]` | Clones an Edgeworker from the existing Edgeworker Id. |
+| list-contracts \| li-contract `[options]` | List of contract ids that user has access to. |
+| list-restier \| li-restier `[options]` | List Resource Tiers that can be used to create or clone EdgeWorker Id. |
+| show-restier \| show-restier `<edgeworker-identifier>` | Customers can get Resource Tier details for a specific EdgeWorker Id. |
```

Comment:
very minor but **show-res-tier** might be easy to understand. same for **list-res-tier**